src/VirtoCommerce.ExperienceApiModule.Web/Module.cs

```diff
@@ -61,6 +64,7 @@ public void Initialize(IServiceCollection services)
 
 
             services.AddSingleton<IStoreCurrencyResolver, StoreCurrencyResolver>();
+            services.TryAddSingleton<Func<IPasswordValidator<ApplicationUser>>>(provider => () => provider.CreateScope().ServiceProvider.GetService<IPasswordValidator<ApplicationUser>>());

```

Comment:
Why do we use factory instead of direct DI?

Reply:
Cause it is scoped service, and I need factory, which is missed

Reply:
`UserManager` already has `PasswordValidators` insde.

Reply:
https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.identity.usermanager-1?view=aspnetcore-5.0#properties

Reply:
So, can you use https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.identity.usermanager-1.validatepasswordasync?view=aspnetcore-5.0#Microsoft_AspNetCore_Identity_UserManager_1_ValidatePasswordAsync__0_System_String_ ?